utils/src/ooinstall/oo_config.py

```diff
@@ -238,6 +252,10 @@ def _read_config(self):
 
                 # Parse the hosts into DTO objects:
                 for host in host_list:
+                    host['other_variables'] = {}
+                    for variable, value in host.iteritems():
+                        if variable not in HOST_VARIABLES_BLACKLIST:
+                            host['other_variables'][variable] = value
```

Comment:
Yes. The reason is that the Host object has a number of attributes (connect_to, public_ip, containerized, etc) and any other variables specified in the YAML file need to go somewhere. In this case, that somewhere is a dictionary called `other_variables`. I suppose we could accomplish something similar by adding attributes dynamically, but I like having them all in once place for easy in writing the ansible inventory later